src/client/components/AddNewGroupComponent/AddNewGroup.stories.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import GroupHandler from './GroupHandler';
+
+export default { title: 'Add new group' };
+
+export const Component = () => <GroupHandler />;
```

Comment:
Create another story when you pass `group` is set to something so that both versions of this component are present.